l10n-pocket/en/pocket/tos.ftl

```diff
@@ -0,0 +1,128 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+## Save to pocket URL: https://dev.tekcopteg.com/tos/
+
+
+pocket-tos-terms-of-service = Terms of Service
+pocket-tos-posted-date = Posted January 27, 2021
+pocket-tos-please-read = Please read the following terms of service ("Agreement" or "Terms of Service") before you install the { -brand-name-pocket } application or use any of the products or services we provide through our application, software or website (all of which are referred to collectively herein as the "{ -brand-name-pocket-technologies }"). The { -brand-name-pocket-technologies } are owned and licensed by { -brand-name-mozilla-corporation }. (referred to herein as "we," "us," "our," "{ -brand-name-mozilla }") and are operated by { -brand-name-mozilla }'s business unit known as "{ -brand-name-mozilla }", which has developed applications and services for use in connection with a variety of devices and platforms, including without limitation, { -brand-name-windows }, { -brand-name-mac-short}, { -brand-name-iphone }, { -brand-name-ipad }, { -brand-name-kindle-fire }, { -brand-name-android }, { -brand-name-blackberry }, { -brand-name-windows } phone, { -brand-name-chrome }, { -brand-name-safari }, { -brand-name-firefox } and other mobile devices and computers. The Terms of Service constitute a binding legal agreement, which govern your use of the { -brand-name-pocket-technologies } via any platform or device. By installing the { -brand-name-pocket } application, visiting our website or installing or using any of the { -brand-name-pocket-technologies }, you are accepting these Terms of Service. If you do not agree to these terms, please do not install our application, access our website or use any of our products or services. This Agreement includes the following provisions, and you can jump to each by selecting the links below:
```

Comment:
**Nit (blocking):** should say `operated by Mozilla's business unit known as "Pocket"`
```suggestion
pocket-tos-please-read = Please read the following terms of service ("Agreement" or "Terms of Service") before you install the { -brand-name-pocket } application or use any of the products or services we provide through our application, software or website (all of which are referred to collectively herein as the "{ -brand-name-pocket-technologies }"). The { -brand-name-pocket-technologies } are owned and licensed by { -brand-name-mozilla-corporation }. (referred to herein as "we," "us," "our," "{ -brand-name-mozilla }") and are operated by { -brand-name-mozilla }'s business unit known as "{ -brand-name-pocket }", which has developed applications and services for use in connection with a variety of devices and platforms, including without limitation, { -brand-name-windows }, { -brand-name-mac-short}, { -brand-name-iphone }, { -brand-name-ipad }, { -brand-name-kindle-fire }, { -brand-name-android }, { -brand-name-blackberry }, { -brand-name-windows } phone, { -brand-name-chrome }, { -brand-name-safari }, { -brand-name-firefox } and other mobile devices and computers. The Terms of Service constitute a binding legal agreement, which govern your use of the { -brand-name-pocket-technologies } via any platform or device. By installing the { -brand-name-pocket } application, visiting our website or installing or using any of the { -brand-name-pocket-technologies }, you are accepting these Terms of Service. If you do not agree to these terms, please do not install our application, access our website or use any of our products or services. This Agreement includes the following provisions, and you can jump to each by selecting the links below:
```